testsuite/MDAnalysisTests/analysis/test_align.py

```diff
@@ -438,31 +445,30 @@ def test_average_structure_deprecated_attrs(self, universe, reference):
     def test_average_structure(self, universe, reference):
         ref, rmsd = _get_aligned_average_positions(self.ref_files, reference)
         avg = align.AverageStructure(universe, reference).run()
-        assert_almost_equal(avg.results.universe.atoms.positions, ref,
-                            decimal=4)
-        assert_almost_equal(avg.results.rmsd, rmsd)
+        assert_allclose(avg.results.universe.atoms.positions, ref, rtol=0, atol=1.5*(1e-4))
+        assert_allclose(avg.results.rmsd, rmsd, rtol=0, atol=1.5*(1e-7))
 
```

Comment:
decimalint, optional
Desired precision, default is 7.

Reply:
Yes, we can explicitly request the former default.